### PR TITLE
chore(flow): refresh setup and respec product roadmap

### PR DESCRIPTION
**Pull request by gmickel**
## Summary

- refresh the repository-local Flow-Next runtime and generated agent setup to v3.13
- reconcile the remaining roadmap against shipped behavior
- close fn-63 as already delivered
- narrow fn-60 and fn-61 to their genuinely missing trust/addressability layers
- redesign fn-71 around secure raster assets in published document shares
- promote fn-86 as a deterministic read-only audit MVP
- retain fn-76 behind an evidence gate
- mark fn-61, fn-60, fn-71, and fn-86 ready from the owner's explicit approval

## Product order

1. fn-61 — durable section targets and citations
2. fn-60 — parser-backed reference-safe refactors
3. fn-71 — bundled images in published document shares
4. fn-86 — read-only knowledge integrity audits

fn-76 remains deferred. fn-63 is closed.

## Verification

- `flowctl validate --all` — valid, 0 errors
- `bun run lint:check` — clean
- `bun test` — 3,619 passed, 2 skipped, 0 failed
- tracker package manifest verified
- every new task investigation target exists

Planning/setup only; no product implementation.

### CODE REVIEW

**Comment by gmickel** [comment]
@codex review

File: .flow/bin/flowctl_tracker/lifecycle/verbs.py [hunk 1]
```diff
@@ -0,0 +1,818 @@
...
+                f"--id {resolved_id!r} does not match the id GraphQL resolves "
+                f"for {identifier!r} ({check['id']!r})",
+                subtype="durable_mismatch",
+                details={"normalized": "durable", "candidates": [
+                    {"durable": resolved_id, "role": "caller"},
+                    {"durable": check["id"], "role": "graphql"},
+                ]})
+        if isinstance(check, TrackerError) and check.cls not in (
+                ErrorClass.TRANSPORT, ErrorClass.RATE_LIMITED):
+            return check
```
[the remark this answers is not in the document]
Comment (gmickel) [reply .flow/bin/flowctl_tracker/lifecycle/verbs.py:749]: Fixed in 8cb33ab0. Caller-supplied MCP durable IDs now treat GraphQL AUTH as an unavailable verification channel, while reachable mismatches and semantic NOT_FOUND failures still fail closed. Verified with an isolated AUTH/TRANSPORT/mismatch/NOT_FOUND matrix and tracker manifest validation.

File: .flow/specs/fn-63-workspace-native-commands-and-agent-callable-actions.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   "plan_review_status": "unknown",
   "plan_reviewed_at": null,
   "spec_path": ".flow/specs/fn-63-workspace-native-commands-and-agent-callable-actions.md",
-  "status": "open",
+  "status": "done",
```
[the remark this answers is not in the document]
Comment (gmickel) [reply .flow/specs/fn-63-workspace-native-commands-and-agent-callable-actions.json:18]: Fixed in 8cb33ab0. The fn-63 children are now serialized as done; the same portability defect affected historical closed specs because runtime-only state masked stale sidecars in existing clones, so I materialized done only for task sidecars whose parent spec is already done. Fresh empty-state : 111 specs, 354 tasks, 0 errors.

**Comment by gmickel** [comment]
@codex review

**Comment by gmickel** [comment]
@codex review

File: .flow/tasks/fn-60-reference-safe-file-operations-and-refactors.1.md [hunk 3]
```diff
@@ -19,9 +19,8 @@ Initial slice:
 - [ ] Tests cover rename/move/reference edge cases
 
 ## Done summary
-
+Reconciled as shipped: shared browser-safe file-operation planners and warning summaries are in main.
```
[the remark this answers is not in the document]
Comment (gmickel) [reply .flow/tasks/fn-60-reference-safe-file-operations-and-refactors.1.md:22]: Fixed in 8d7ad868. I completed all eight reconciled tasks through flowctl and serialized their structured sidecars. Ready-set proof now exposes only fn-60.5 and fn-61.5; fresh empty-state validation remains 111 specs / 354 tasks / 0 errors. Full gate: lint clean, 3,619 pass, 2 platform skips, 0 fail.

**Comment by gmickel** [comment]
@codex review
